lib/paths.cf

```diff
@@ -315,7 +315,7 @@ bundle common paths
       "path[tr]"            string => "/usr/bin/tr";
       "path[wc]"            string => "/usr/bin/wc";
       "path[wget]"          string => "/usr/bin/wget";
-      "path[realpath]"      string => "/usr/bin/realpath";
+      "path[realpath]"      string => "/usr/bin/readlink -e";
```

Comment:
Yes it's nice in that regard.

I wonder of it would make sense if paths were imolemted at the command line to allow a paths.path to suffice for a first hit. 

Then the paths bundle could be a user addition with these kinds of overrides.